### DISCUSSION TITLE
Tickets don't seem to submit.

### DISCUSSION THREAD

**Discussion by mpmrepository** [vercel/community#2101] [Bug Report]
I've submitted multiple tickets now on https://vercel.com/help. It says the ticket was submitted but I've not received any email to confirm and in the console i get a 400 error:

`code
: 
"cannot_reactivate"
message
: 
"Cannot retrieve members to reactivate on a team that is not on the hobby plan."`

I've tried submitting on multiple accounts. The issue is in relation to a Pro account

**Comment by amyegan** [comment]
Thanks for reporting this, @mpmrepository. I reached out to the team about the error. 

In the meantime, it looks like you do have an open case. Multiple cases were opened and have since been merged into one, so you should have an email notification about that. If the email didn't make it to your inbox, please check your spam or junk folder and adjust filter preferences as needed.
